apis/routes/userRouter.js

```diff
@@ -0,0 +1,8 @@
+const router = require("express").Router();
+
+const { userController } = require("../controllers");
+
+router.route("/signin").post(userController.makeTokenAsCode);
```

Comment:
```suggestion
router.route("/kakao-signin").post(userController.makeTokenAsCode);
```
지금 기획으로는 일반로그인이 없다고 하더라도 확장성이 고려된 endpoint 작성을 지향해주세요